ports/servoshell/rendering_context.rs

```diff
@@ -0,0 +1,297 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's keep the implementation of this `RenderingContext` the `webrender` crate and make it the default implementation, otherwise it is going to be really annoying for all other embedders to embed Servo without a lot of code duplication.